frontend/src/components/pages/ContactUs/contactus.js

```diff
@@ -1,8 +1,13 @@
-import React from 'react'
+import React from 'react';
+import Navbar from '../../navbar/navbar.js';
+
 
 function ContactUs (){
```

Comment:
Cuidado con los espaciados, (creo que el linter te debe estar marcando esta línea):
```suggestion
function ContactUs() {
```